package.json

```diff
@@ -18,7 +19,8 @@
     "heroku-client": "^1.9.1",
     "isomorphic-fetch": "^2.0.0",
     "origami-build-tools": "^2.13.0",
-    "semver": "^4.3.1"
+    "semver": "^4.3.1",
+    "fastly": "git+https://github.com/Financial-Times/fastly.git#v2.5.0"
```

Comment:
Thanks will fix.  Also,  according to npm-check I can get rid of debug and heroku-client.  does that sound right?  I know you hate debug and I guess heroku-client is a dep of haikro anyway